scripts/utils/createJestConfig.js

```diff
@@ -25,48 +25,36 @@ module.exports = (resolve, rootDir, isEjecting) => {
       'src/**/*.{js,jsx,ts,tsx}',
       '!**/*.d.ts'
     ],
-    setupFiles: [resolve('config/polyfills.js')],
+    setupFiles: ['react-app-polyfill/jsdom'],
     setupTestFrameworkScriptFile: setupTestsFile,
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.(j|t)s?(x)',
       '<rootDir>/src/**/?(*.)(spec|test).(j|t)s?(x)',
     ],
-    testEnvironment: 'node',
+    testEnvironment: 'jsdom',
     testURL: 'http://localhost',
     transform: {
-      '^.+\\.(js|jsx|mjs)$': isEjecting
+      '^.+\\.(js|jsx)$': isEjecting
         ? '<rootDir>/node_modules/babel-jest'
         : resolve('config/jest/babelTransform.js'),
       '^.+\\.tsx?$': resolve('config/jest/typescriptTransform.js'),
       '^.+\\.css$': resolve('config/jest/cssTransform.js'),
-      '^(?!.*\\.(js|jsx|mjs|css|json)$)': resolve(
-        'config/jest/fileTransform.js'
-      ),
+      '^(?!.*\\.(js|jsx|css|json)$)': resolve('config/jest/fileTransform.js'),
     },
     transformIgnorePatterns: [
-      '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$',
+      '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$',
+      '^.+\\.module\\.(css|sass|scss)$',
     ],
     moduleNameMapper: {
       '^react-native$': 'react-native-web',
+      '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
     },
-    moduleFileExtensions: [
-      'web.ts',
-      'ts',
-      'web.tsx',
-      'tsx',
-      'web.js',
-      'js',
-      'web.jsx',
-      'jsx',
-      'json',
-      'node',
-      'mjs',
-    ],
     globals: {
       'ts-jest': {
         tsConfigFile: paths.appTsTestConfig,
```

Comment:
When you will update `ts-jest`, you need to update this line to
```json
"tsConfig": paths.appTsTestConfig
```
Docs: https://github.com/kulshekhar/ts-jest/blob/2852078d83cc58eed3986aa2e3e210cd5d306b23/docs/user/config/tsConfig.md#path-to-a-tsconfig-file